### PR TITLE
ST-584 Send Order Select Recipients

### PR DESCRIPTION
**Pull request by ElifDenizEsea**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines
- [x] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/ST-584

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/sptribs/caseworker/event/ReinstateCase.java [hunk 1]
```diff
@@ -80,12 +80,15 @@ public SubmittedCallbackResponse reinstated(CaseDetails<CaseData, State> details
         messageLine2.append("%n##  A notification will be sent via email to: ");
         if (!CollectionUtils.isEmpty(cicCase.getNotifyPartySubject())) {
             messageLine2.append("Subject, ");
+            cicCase.setNotifyPartySubject(null);
         }
```
Comment (osheaf01) [inline src/main/java/uk/gov/hmcts/sptribs/caseworker/event/ReinstateCase.java:84]: Just curious - if it's not empty, you append something, then you set it to null - why?

Comment (ElifDenizEsea) [reply src/main/java/uk/gov/hmcts/sptribs/caseworker/event/ReinstateCase.java:84]: for using same objects in other notify recipient actions, like send order

**Review by osheaf01 — DISMISSED** [review]
LGTM

**Review by sjamihmcts — APPROVED** [review]

**Review by sjamihmcts — APPROVED** [review]

**Review by osheaf01 — APPROVED** [review]
LGTM

**Pull request merged**